### DISCUSSION TITLE
New UI unable to click on preview, and image disappears when gen finishes from ui.

### DISCUSSION THREAD

**Discussion by Weatherby43** [lllyasviel/stable-diffusion-webui-forge#870] [Q&A]
version: [f1.0.0v1.10.0rc-previous-14-g33e381f1](https://github.com/lllyasviel/stable-diffusion-webui-forge/commit/33e381f16a24920abaeecc70216ded4339a082ee)  •  python: 3.10.9  •  torch: 2.1.2+cu121  •  xformers: 0.0.23.post1  •  gradio: 4.39.0  •  checkpoint: [ff3ed826b3](https://google.com/search?q=ff3ed826b3a7f4da6a0b4e4bdaf05e13411ac631cfb300a6bb03ce2c93d7e8e7)

I updated Forge by cloning the repo, I ran the webui-user.bat file, then, once it finished, closed it and then ran git pull  to pull the newest updates. I got the above versions, which I believe means the most current experimental version of Forge. 

Now, however, on this new version, the preview window is acting unusual. Instead of it showing a rolling preview of the image being generated every N steps and you can click on it to blow it up, it no longer is clickable while forming the previews before the gen is done. Once the gen finishes, the preview window goes to a broken image http icon and when you click on it, it blows up this icon and will not show the finished image. Going into the output directory, the image is there just fine. Also, this impacts the 'run highres fix' button as well, as if you click the button, it returns a 'none value' error. So, I either have to run all images with highres fix enabled, or don't use it at all. 

Also, for some reason the new version of the UI doesn't pull values from the config.jason or ui-config.jason file. I've adjusted the ui-config.jason as I have in past versions to contain my default neg prompt and steps, sampler etc. Every time I open the new version it resets those values to the initial values.

Anyone else noticing this?

**Comment by zeittresor** [comment]
i have mostly the same issue here. zooming to fullscreen only works just directly after click on the generate button, not if i wait too long. if i close the fullscreen view i can not reopen it (nothing happens if i click on the regular preview). Also custom changes like resolution or sampler to the ui-config.json are irgnored by the webui..

**Comment by Enferlain** [comment]
Any updates? Just installed and same

**Comment by offroadguy56** [comment]
Awkward late reply here. I had the same problem. However for me, a fresh install was working fine but after getting it setup with all my settings the final image stopped showing. In progress previews were working fine and i could open the image just fine on disk too.

The problem for me was I was saving the images direct to a network SMB share. `\\SMB-Share\forge-neo\txt2img-images`. For some reason Forge does not like that. 

What you need to do to fix my problem is map the network share to a drive letter. You can right click the root output folder and click map network drive. Then set up your pathing in the settings like usual.
